src/build.rs

```diff
@@ -232,7 +232,7 @@ fn build_implib_file(
             dlltool_command.arg("-D").arg(format!("{}.dll", name));
```

Comment:
Prefix also the DLL to match Autotools/CMake/Meson behaviour?

Reply:
I tried to, but for some reason It failed to find the dll file when installing.

Reply:
Perhaps because it needs to be paired w/ the change below as well?

Reply:
I tried that too, You could try it yourself. Patch cargo-c As I did and see if it works for you.